windows/src/developer/TIKE/http/Keyman.Developer.System.HttpServer.Debugger.pas

```diff
@@ -263,6 +294,23 @@ procedure TDebuggerHttpResponder.ProcessRequest(AContext: TIdContext;
       FKeyboardsCS.Leave;
     end;
 
+    FModelsCS.Enter;
+    try
+      for model in FModels.Keys do
+      begin
+        src := ChangeFileExt(model, '');
+        response := response + Format(
+          '  keyman.modelManager.register({'#13#10+
+          '    "id": "%s", '#13#10+
+          '    "languages": ["en"],'#13#10+
```

Comment:
Why is this registering`["en"]`? Won't it need to match the language ID for the model?

Reply:
For simplicity (and because we don't need to register by language), all keyboards and models are registered under `'en'` in the debugger. This does prod me to remember that we need a way to swap models if more than one is loaded in the debugger... (consider that a to-do item to come in a follow-up PR)